install/kubernetes/cilium/README.md

```diff
@@ -60,13 +60,16 @@ contributors across the globe, there is almost always someone available to help.
 | aksbyocni.enabled | bool | `false` | Enable AKS BYOCNI integration. Note that this is incompatible with AKS clusters not created in BYOCNI mode: use Azure integration (`azure.enabled`) instead. |
 | alibabacloud.enabled | bool | `false` | Enable AlibabaCloud ENI integration |
 | annotateK8sNode | bool | `false` | Annotate k8s node upon initialization with Cilium's metadata. |
-| auth.mTLS.enabled | bool | `false` | Enable mtls-spiffe authentication method in CiliumNetworkPolicy |
-| auth.mTLS.port | int | `4250` | port on the agent which is used to mTLS handshakes on |
-| auth.mTLS.spiffeTrustDomain | string | `"spiffe.cilium.io"` | SPIFFE trust domain to use for fetching certificates |
-| auth.mTLS.spireAdminSocketPath | string | `"/run/spire/sockets/admin.sock"` | SPIRE socket path where the SPIRE delegated api agent is listening |
-| auth.mTLS.spireAgentSocketPath | string | `"/run/spire/sockets/agent/agent.sock"` | SPIRE agent socket path where the SPIRE agent is listening |
-| auth.mTLS.spireServerAddress | string | `"spire-server.spire.svc.cluster.local:8081"` | SPIRE server endpoint This endpoint will be automatically injected later once embedded SPIRE installation is done. |
-| auth.mTLS.spireServerConnectionTimeout | string | `"10s"` | SPIRE server connection timeout |
+| auth.mTLS.port | int | `4250` | Port on the agent which is used to mTLS handshakes on |
+| auth.mTLS.spire.adminSocketPath | string | `"/run/spire/sockets/admin.sock"` | SPIRE socket path where the SPIRE delegated api agent is listening |
+| auth.mTLS.spire.agentSocketPath | string | `"/run/spire/sockets/agent/agent.sock"` | SPIRE socket path where the SPIRE workload agent is listening Applies to both the Cilium Agent and Operator |
+| auth.mTLS.spire.connectionTimeout | string | `"30s"` | SPIRE connection timeout |
+| auth.mTLS.spire.enabled | bool | `false` | Enable SPIRE integration |
+| auth.mTLS.spire.identities | object | `{"ciliumAgent":"cilium-agent","ciliumOperator":"cilium-operator","delegatedClient":"dclient"}` | Identity paths for the SPIFFE URLs, without suffix / SPIFFE URLs will look like spiffe://trust-domain/identity |
+| auth.mTLS.spire.install | object | `{"enabled":false,"namespace":"cilium-spire"}` | Settings to control the SPIRE installation and configuration |
+| auth.mTLS.spire.install.enabled | bool | `false` | Enable SPIRE installation Note that this will only take effect if auth.mTLS.enabled is true |
```

Comment:
Clarity nit
```suggestion
| auth.mTLS.spire.install.enabled | bool | `false` | Enable SPIRE installation; this will only take effect if auth.mTLS.enabled is true |
```